ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10-upgrade-from-eus-4.8.yaml

```diff
@@ -29,15 +29,8 @@ tests:
       BASE_DOMAIN: aws-2.ci.openshift.org
       E2E_RUN_INTERMEDIATE_TAGS: '@4.9 and @aws-ipi and @upgrade-sanity'
       E2E_RUN_LATEST_TAGS: '@4.10 and @aws-ipi and @upgrade-sanity'
-      E2E_SKIP_TAGS: not @console and not @destructive and not @disconnected and not
-        @flaky and not @inactive and not @network-multitenant and not @network-ovnkubernetes
-        and not @prod-only and not @proxy and not @stage-only and not @upgrade-check
-        and not @upgrade-prepare
       UPGRADE_CHECK_RUN_TAGS: '@4.8 and @aws-ipi and @upgrade-check'
       UPGRADE_PRE_RUN_TAGS: '@4.8 and @aws-ipi and @upgrade-prepare'
-      UPGRADE_SKIP_TAGS: not @console and not @destructive and not @disconnected and
-        not @flaky and not @inactive and not @prod-only and not @proxy and not @qeci
-        and not @stage-only
```

Comment:
`not @upgrade-check and not @upgrade-prepare` can also be added here because they are not meant for the E2E.

Reply:
The `UPGRADE_SKIP_TAGS` is used for `cucushift-upgrade-pre`, so `not @upgrade-check and not @upgrade-prepare` should not be include there.

If you mean to add them for `E2E_SKIP_TAGS`, then they are added in the default for `E2E_SKIP_TAGS`.

Reply:
You are right, thank you